front-end/src/renderer/pages/TransactionDetails/TransactionDetails.vue

```diff
@@ -629,7 +630,9 @@ const cancel = 'Cancel';
                     "
                   >
                     <h4 :class="detailItemLabelClass">Description</h4>
-                    <p :class="detailItemValueClass" data-testid="p-description-field">
+                    <p :class="detailItemValueClass" data-testid="p-description-field"
+                       style="white-space: normal; word-break: break-word; overflow-wrap: break-word;"
```

Comment:
Inline styles are bad practice, consider adding the `text-break` to the `detailItemValueClass` and remove the inline styles